src/shepherds-pie.md

```diff
@@ -6,7 +6,7 @@
 
 ## Ingredients
 
-- 600 gramms (g) of mealy/floury potatoes
+- 600 grams (g) of mealy/floury potatoes
```

Comment:
typo fix

Reply:
What is this? It was already fixed in commit 901a610fc9df4d341bd2d29cf96ed68b06d548a1

Reply:
Maybe I don't understand GitHub. Did someone else have to approve that commit because the Reviewer was the one who co-authored?

Reply:
nah, looks like it is fixed. Not sure why it still showed this for @DGKSK8LIFE